File: regression2/e2epulloriginlag1/1728291164201/pipelines/Pipeline-Automation-PULL_ORIGIN1-1728291126120/code/src/main/scala/io/prophecy/pipelines/pipelineautomationpull_origin11728291126120/graph/Reformat_1.scala
package io.prophecy.pipelines.pipelineautomationpull_origin11728291126120.graph

import io.prophecy.libs._
import io.prophecy.pipelines.pipelineautomationpull_origin11728291126120.functions.PipelineInitCode._
import io.prophecy.pipelines.pipelineautomationpull_origin11728291126120.functions.UDFs._
import io.prophecy.pipelines.pipelineautomationpull_origin11728291126120.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._
object Reformat_1 { def apply(context: Context, in: DataFrame): DataFrame = in }

File: test/ayushdev/nonmainismain/pipelines/TESTPipelineNonMain/code/testpipelinenonmain/graph/Reformat_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from testpipelinenonmain.config.ConfigStore import *
from testpipelinenonmain.udfs.UDFs import *

def Reformat_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0
